sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3078,9 +3078,8 @@ object functions {
    * string.
    *
    * @param e a string column containing JSON data.
-   * @param schema the schema to use when parsing the json string as a json string. In Spark 2.1,
-   *               the user-provided schema has to be in JSON format. Since Spark 2.2, the DDL
-   *               format is also supported for the schema.
+   * @param schema the schema to use when parsing the json string as a json string
+   *               or a DDL-formatted string.
```

Comment:
(These are strictly not related with this PR but i just decided to pick this up.)